tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetCore5AutoUserEvents.cs

```diff
@@ -27,6 +32,9 @@ protected AspNetCore5AutoUserEvents(AspNetCoreTestFixture fixture, ITestOutputHe
             _enableSecurity = enableSecurity;
             _fixture.SetOutput(outputHelper);
             EnableRasp(false);
+            // without this, the developer exception page intercepts our blocking middleware and doesn't let us write the proper response
```

Comment:
Is this a "real" problem we need to solve 🤔

Reply:
I dont think it s a real problem as in "development" they will want to see the developer page so it doesn't matter if the Response is not rewrittten.. But for integration tests, we want the verify files to be as in production